Runtime/AvatarCreator/UI/SelfieElement.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public class SelfieElement : MonoBehaviour
+    {
+        private const string TAG = nameof(SelfieElement);
+
+        [SerializeField] private ImageConfirmationElement imageConfirmationElement;
+        [SerializeField] private BodyType bodyType;
+        [SerializeField] private OutfitGender gender;
+
+        [Space(5)]
+        [Header("Events")]
+        public UnityEvent<GameObject,AvatarProperties> onAvatarCreated;
+
+        private void OnEnable()
+        {
+            imageConfirmationElement.onImageConfirmed.AddListener(OnPhotoCaptured);
+        }
+
+        private void OnDisable()
+        {
+            imageConfirmationElement.onImageConfirmed.RemoveListener(OnPhotoCaptured);
+        }
+
+        private async void OnPhotoCaptured(Texture2D texture)
```

Comment:
I think we should remove the imageConfirmationElement reference (dependency) and have some public function like LoadAvatarFromSelfie (or something similar) here instead. Then the function can just be assigned in the prefab itself. 
By doing this they can use this selfie element in combination with not only our other elements but also their own scripts without having to use our ImageConfirmation or PhotoCapture element. 
 EG if they have a native camera/file picker they can just pass the image to Load from selfie etc.